.github/workflows/black.yml

```
@@ -9,21 +9,29 @@
-        uses: peter-evans/create-pull-request@v3
+        if: ${{ failure() && steps.black.outputs.is_formatted == 'true' }}
+        uses: peter-evans/create-pull-request@v5
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "Apply black formatting"
           title: "Format code with black"
           body: "This pull request updates the code to conform to the black formatting standard."
           branch: "black-formatted"
-          draft: false
+          base: ${{ github.ref }}
+          delete-branch: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)